### ISSUE TITLE
Feature request: resize node viewing pane

### ISSUE THREAD

**Issue by michael-bronte** [neo4j-apps/neo4j-bloom#67]
If users can resize the node/link viewer pane this would enable Bloom to be used as a DIY intelligence (like) system as more complex key value documents can be displayed. At least, our organisation could definitely use Bloom a lot more with this feature.

Also, the size adjustments should be persisted in the user's browser (or similar) so they only have to make them once.

I added a screenshot to elaborate on this.
<img width="510" alt="send_neo4j" src="https://github.com/neo4j-devtools/neo4j-bloom/assets/91190098/7c64efeb-2708-4c0e-a182-4a7607d5d934">

Another nice to have would be enabling hyperlinks to display different text (to the URL)

Thanks!

**Comment by joknelid** [comment]
Hi! 

Thanks for the feedback, appreciate the clarity of the screenshot. We have added this to the backlog and it will be reviewed for inclusion in future releases.

There is a somewhat similar feature we've had on the list for a while which is a table view of the graph similar to a spreadsheet. Would this also solve your use case you mention?

**Comment by adriens** [comment]
> Another nice to have would be enabling hyperlinks to display different text (to the URL)

Yes, please, hyperlinks as for now, we have to use Graph XR to efficiently use this kind of UX :pray:

**Comment by michael-bronte** [comment]
Hi Jens,

The use case is more along the lines of* 'an analyst can select a single
node/link and view its associated document'. *That document could be the
nodes/links properties.

Thanks!

**Comment by joknelid** [comment]
Thanks Michael, so it's more about being able to view more property data than just being able to resize the columns?

And noted about the hyperlinks as well!

**Comment by michael-bronte** [comment]
correct, thanks
